### PR TITLE
refactor: Upgrade pg-promise, pg-monitor

### PR DESCRIPTION
**Pull request by vitaly-t**
### New Pull Request Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [ ] I am creating this PR in reference to an [issue](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).

### Issue Description

Upgrades `pg-promise` + `pg-monitor` (they are interdependent, need to be upgraded at once)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,8 +48,8 @@
     "mongodb": "4.10.0",
     "mustache": "4.2.0",
     "parse": "3.4.2",
-    "pg-monitor": "1.5.0",
-    "pg-promise": "10.12.1",
+    "pg-monitor": "2.0.0",
+    "pg-promise": "11.0.0",
```
Comment (dplewis) [inline package.json]: Can you run `npm install` to update the package-lock.json?

**Review by dplewis — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (vitaly-t) [reply package.json]: done

**Comment by mtrezza** [comment]
This includes 2 major version upgrades; is this a breaking change for developers?

**Comment by vitaly-t** [comment]
> This includes 2 major version upgrades; is this a breaking change for developers?

Breaking changes in the library, as per [v11 notes](https://github.com/vitaly-t/pg-promise/releases/tag/11.0.0), but not for parse-server developers or users (at least shouldn't be).

**Comment by mtrezza** [comment]
Thanks for confirming; kindly request a review once this is ready.

**Comment by vitaly-t** [comment]
For some reasons, I no longer can install dependencies, getting this error:

image

Any idea what might be causing the problem?

**Comment by mtrezza** [comment]
Maybe related to your GitHub auth?

**Pull request closed without merging**